structuredheadings/styles/numbering.css

```diff
@@ -178,3 +178,20 @@ body {
 .autonumber.autonumber-5.autonumber-r:before {
     content: counter(autonumber-5, lower-roman) ". ";
 }
+
+/*
+styles for List: 1. 1. 1.
+inspired by these:
+https://stackoverflow.com/questions/4098195/can-ordered-list-produce-result-that-looks-like-1-1-1-2-1-3-instead-of-just-1
+https://stackoverflow.com/questions/10405945/html-ordered-list-1-1-1-2-nested-counters-and-scope-not-working/10405962
+http://jsfiddle.net/qGCUk/4/
+*/
```

Comment:
I want to keep it at the place here, to save time for googling when something would be broken